cypress/support/commands.js

```diff
@@ -0,0 +1,35 @@
+Cypress.Commands.add('clickLink', (selector, linkText) => {
+    cy.get(selector).contains(linkText).click();
+})
+
+Cypress.Commands.add('clickOption', (option) => {
+    cy.get('.tt-highlight').eq(option).click();
+})
+
+Cypress.Commands.add('typeInSearch', (text) => {
+    cy.get('#q')
+      .clear()
+      .type(text)
+})
+
+Cypress.Commands.add('verifyText', (tag, text) => {
+    cy.get(tag).should('contain', text)
+})
```

Comment:
Instead of passing selectors to the custom command, please consider creating a more specific command and using [child commands](https://docs.cypress.io/api/cypress-api/custom-commands#Child-Commands).